components/style/values/computed/text.rs

```diff
@@ -151,3 +151,23 @@ impl TextDecorationsInEffect {
 
 /// A specified value for the `-moz-tab-size` property.
 pub type MozTabSize = GenericMozTabSize<NonNegativeNumber, NonNegativeLength>;
+
+/// computed value for the text-emphasis-style property
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToCss)]
+pub enum TextEmphasisStyle {
+    /// Keyword value for the text-emphasis-style property (`filled` `open`)
+    Keyword(KeywordValue),
+    /// `none`
+    None,
+    /// String (will be used only first character) for the text-emphasis-style property
```

Comment:
"character" is a bit ambiguous here, because "character" in CSS may have different meaning than that in Rust. It should be "grapheme cluster" instead for better exactness. One may think we can use `char` instead if there is only one "character" being used, which is not true, because a grapheme cluster can well include multiple `char`s.

Reply:
Thanks for the explanation